patches/patch_template.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/src/proto_genesis_babylonnet/lib_protocol/data.ml b/src/proto_genesis_babylonnet/lib_protocol/data.ml
+index 061e8c8..f89b553 100644
+--- a/src/proto_genesis_babylonnet/lib_protocol/data.ml
++++ b/src/proto_genesis_babylonnet/lib_protocol/data.ml
+@@ -102,7 +102,7 @@ module Pubkey = struct
+ 
+   let default =
+     Signature.Public_key.of_b58check_exn
+-      "edpkugeDwmwuwyyD3Q5enapgEYDxZLtEUFFSrvVwXASQMVEqsvTqWu"
```

Comment:
can we not patch tezos to make this read from an env variable?

Reply:
I'm not very familiar with OCaml much, but I think it's doable, though, it will require some additional time

Reply:
Anyway, it will require modifying source code in some way

Reply:
@rvem can you check whether it has been discussed somewhere already (among Tezos developers) and create an issue if you don't find one?